lib/colonel_kurtz/validatable.ex

```diff
@@ -0,0 +1,35 @@
+defprotocol ColonelKurtz.Validatable do
```

Comment:
I renamed this from `ValidatableBlock` to `Validatable` since it is also used on the `BlockType.Content` schema.